packages/inter-protocol/test/smartWallet/test-psm-integration.js

```diff
@@ -169,14 +169,16 @@ test('want stable (insufficient funds)', async t => {
     callPipe: [['makeWantMintedInvitation']],
   };
 
-  await offersFacet.executeOffer({
-    id: 'insufficientFunds',
-    invitationSpec,
-    proposal: {
-      give: { In: anchor.make(anchorFunding * 2n) }, // twice the available funds
-      want: {},
-    },
-  });
+  await t.throwsAsync(
```

Comment:
Why are some of the `t.throwsAsync()` calls missing a `message` to match?

Reply:
I started adding them after a bit. The errors messages are tested by the vstorage check.

If we commit to this throing from executeOffer, I can take out the redundant vstorage checks and have a test focused on vstorage writing out when there's an error. The different errors needn't all be checked in vstorage.